pkg/odo/cli/preference/view.go

```diff
@@ -68,6 +68,7 @@ func (o *ViewOptions) Run() (err error) {
 	fmt.Fprintln(w, "Experimental", "\t", showBlankIfNil(cfg.OdoSettings.Experimental))
 	fmt.Fprintln(w, "PushTarget", "\t", showBlankIfNil(cfg.OdoSettings.PushTarget))
 	fmt.Fprintln(w, "Ephemeral", "\t", showBlankIfNil(cfg.OdoSettings.Ephemeral))
+	fmt.Fprintln(w, "ConsentTelemetry", "\t", showBlankIfNil(cfg.OdoSettings.ConsentTelemetry))
```

Comment:
Could we change it from`ConsetTelemetry` to `SendTelemetry`? Maybe it is just me, but it sounds a little bit better 😇

Reply:
`ConsentTelemetry` did seem intuitive to me at first, but I guess `SendTelemetry` makes sense and sounds good too. I'll change that!

Reply:
Now I just realized that it might be good to use the same name as CRC.
I just checked that they have `ConsentTelemetry` 😇 So lets keep it :-)